device_calendar/lib/src/device_calendar.dart

```diff
@@ -61,14 +60,14 @@ class DeviceCalendarPlugin {
   ///
   /// Returns a [Result] containing a list of device [Calendar]
   Future<Result<List<Calendar>>> retrieveCalendars() async {
```

Comment:
Should this return UnmodifiableListView instead of List - https://api.dartlang.org/stable/2.4.1/dart-collection/UnmodifiableListView-class.html

Reply:
ah yep, thanks